src/entities/ether.ts

```diff
@@ -2,26 +2,28 @@ import invariant from 'tiny-invariant'
 import { Currency } from './currency'
 import { NativeCurrency } from './nativeCurrency'
 import { Token } from './token'
-import { WETH9 } from './weth9'
 
 /**
  * Ether is the main usage of a 'native' currency, i.e. for Ethereum mainnet and all testnets
  */
 export class Ether extends NativeCurrency {
-  protected constructor(chainId: number) {
+  private _address: string
```

Comment:
@moodysalem should I move `address` to `BaseCurrency`?

Reply:
i would actually not name this address, `wrappedEtherAddress` or something more descriptive would be better imo